docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,384 @@
+`updateIncidentJson`:: A stringified JSON with mustache variables that corresponds to *Update Case JSON*.
+`createCommentUrl`:: A URL string that corresponds to *Create Comment URL*.
+`createCommentMethod`:: A string that corresponds to *Create Comment Method*.
+`createCommentJson`:: A stringified JSON with mustache variables that corresponds to *Create Comment JSON*.
+
+
+Secrets defines sensitive information for the connector type.
+
+`user`:: A string that corresponds to *User*. Required if `hasAuth` is set to `true`.
+`password`:: A string that corresponds to *Password*. Required if `hasAuth` is set to `true`.
+
+[float]
+[[define-cases-webhook-ui]]
+=== Define connector in Stack Management
+
+Define Webhook - Case Management connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]
+
+Test Webhook - Case Management action parameters.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-test.gif[Webhook - Case Management params test]
+
+[float]
+[[cases-webhook-action-configuration]]
+=== Action configuration
+
+Webhook - Case Management actions have the following configuration properties.
+
+Title:: A title for the issue, used for searching the contents of the knowledge base.
+Description:: The details about the incident.
+Labels:: The labels for the incident.
+Additional comments:: Additional information for the client, such as how to troubleshoot the issue.
+
+
+[float]
+[[cases-webhook-connector-full-example]]
+== Full Example with Third Party System
+
+The following is an example of using the Webhook - Case Management Connector with a real third party case management system, Jira. We will reference their https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/[API documentation] for creating an issue and use a demo instance of Jira with real working endpoints.
+
+_Note: We are using Jira as an example third party. However, if you wish to connect with Jira, using the <<jira-action-type, preconfigured Jira connector>> will simplify your setup._
+
+[float]
+====  Step 1 - Set up connector
+
+In the Webhook - Case Management connector create flyout, we begin by entering a *Name* for the connector - `Jira Test Connector`. We are going to use basic authentication in this example, so we keep the *Require authentication* checkbox checked and enter the *Username* and *Password* for the test instance, `test-user@elastic.co` and `notarealpassword`. We will not be setting any *Headers* for the requests.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-step1.png[Webhook - Case Management connector Step 1, Jira example]
+
+[float]
+====  Step 2 - Create case
+
+Taking a look at the Jira https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-post[Create issue method documentation], we can find what values we will need to fill step 2.
```

Comment:
```suggestion
To find the required values for this step, refer to the {jira} https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-post[Create issue method documentation].
```

Reply:
It might also be helpful to point towards where the information is in the doc. That way user aren't immediately able to ID what they should be looking for. :)

Also, I like what you're doing with walking users though each step in the example. One way we could continue to do that here is to describe the purpose of this step. Is it to create a case record in JIRA once the connector is created? Or is it to set up the connector so that a case is created in JIRA once users push changes to JIRA? Or is it something entirely different?